### ISSUE TITLE
The /oembed endpoint returns 500 response code

### ISSUE THREAD

**Issue by vykulakov** [alerta/alerta#861]
**Describe the bug**
When I try to open the following url `https://example.com/oembed?url=https%3A%2F%2Fexample.com%2Falerts%2Fcount%3Fstatus%3Dopen&maxwidth=200&maxheight=200&api-key=xxx&title=Mobile%20Service&callback=jQuery213019287837023337473_1551364356555&_=1551364356556` it always returns 500 response code with the following body:
```
{
"code": 500,
"errors": [
"Traceback (most recent call last):\n  File \"/usr/local/lib/python3.5/dist-packages/flask/app.py\", line 1813, in full_dispatch_request\n    rv = self.dispatch_request()\n  File \"/usr/local/lib/python3.5/dist-packages/flask/app.py\", line 1799, in dispatch_request\n    return self.view_functions[rule.endpoint](**req.view_args)\n  File \"/usr/local/lib/python3.5/dist-packages/flask_cors/decorator.py\", line 128, in wrapped_function\n    resp = make_response(f(*args, **kwargs))\n  File \"/usr/local/lib/python3.5/dist-packages/alerta/auth/decorators.py\", line 49, in wrapped\n    return f(*args, **kwargs)\n  File \"/usr/local/lib/python3.5/dist-packages/alerta/utils/response.py\", line 14, in decorated\n    data = str(func(*args, **kwargs).data)\nAttributeError: 'tuple' object has no attribute 'data'\n"
],
"message": "'tuple' object has no attribute 'data'",
"status": "error"
}
```
In the Alerta log at the same time:
```
Feb 28 14:35:29 localhost alertad[30345]: ERROR 'tuple' object has no attribute 'data'
Feb 28 14:35:29 localhost alertad[30345]: Traceback (most recent call last):
Feb 28 14:35:29 localhost alertad[30345]:   File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1813, in full_dispatch_request
Feb 28 14:35:29 localhost alertad[30345]:     rv = self.dispatch_request()
Feb 28 14:35:29 localhost alertad[30345]:   File "/usr/local/lib/python3.5/dist-packages/flask/app.py", line 1799, in dispatch_request
Feb 28 14:35:29 localhost alertad[30345]:     return self.view_functions[rule.endpoint](**req.view_args)
Feb 28 14:35:29 localhost alertad[30345]:   File "/usr/local/lib/python3.5/dist-packages/flask_cors/decorator.py", line 128, in wrapped_function
Feb 28 14:35:29 localhost alertad[30345]:     resp = make_response(f(*args, **kwargs))
Feb 28 14:35:29 localhost alertad[30345]:   File "/usr/local/lib/python3.5/dist-packages/alerta/auth/decorators.py", line 49, in wrapped
Feb 28 14:35:29 localhost alertad[30345]:     return f(*args, **kwargs)
Feb 28 14:35:29 localhost alertad[30345]:   File "/usr/local/lib/python3.5/dist-packages/alerta/utils/response.py", line 14, in decorated
Feb 28 14:35:29 localhost alertad[30345]:     data = str(func(*args, **kwargs).data)
Feb 28 14:35:29 localhost alertad[30345]: AttributeError: 'tuple' object has no attribute 'data'
```

**To Reproduce**
1. Invoke `/oembed` with correct parameters
2. See error

**Expected behavior**
I expect a regular response.

**Environment:**
 - OS: Debian 9.7
 - Version:
    - alerta 6.7.5
    - alerta client 6.5.0
    - requests 2.21.0
    - click 7.0

**Additional context**
Alertad config file:
```
$ cat /etc/alertad.conf
DEBUG = False
USE_PROXYFIX = True
ALLOWED_ENVIRONMENTS = ['Production', 'Development', 'Testing']

LOG_HANDLERS = ['console']
LOG_FILE = '/var/log/alertad.log'
LOG_MAX_BYTES = 5*1024*1024  # 5 MB
LOG_BACKUP_COUNT = 2
LOG_FORMAT = 'simple'

DATABASE_URL = 'postgresql://correct_url'
DATABASE_NAME = 'alerta'

CORS_ORIGINS = [
    'correct_origin'
]

ADMIN_USERS = ['admin']

AUTH_REQUIRED = True
AUTH_PROVIDER = 'gitlab'
GITLAB_URL = 'gitlab_url'
OAUTH2_CLIENT_ID = 'client_id'
OAUTH2_CLIENT_SECRET = 'client_secret'
ALLOWED_GITLAB_GROUPS = ['correct_group']

ALERT_TIMEOUT = 300
```

**Comment by satterly** [comment]
Fixed by #853

**Comment by satterly** [comment]
Duplicate of #860
